docs/src/pages/components/text/react.mdx

```diff
@@ -97,19 +97,10 @@ import '@aws-amplify/ui-react/styles.css';
 
 <Example>
   <Text className="primary-styled-variable-text" variation="primary">
-    This is my styled text
+    {'This is my styled text'}
   </Text>
 </Example>
 
-To replace the Text styling, unset it:
```

Comment:
Removing this guidance because using `all: unset` doesn't have the effect that most developers would expect, and would cause more harm than good.